src/lib/cache.js

```diff
@@ -67,10 +67,17 @@ export default {
   get: key => {
     return new Promise((resolve, reject) => {
       if (isNull(client)) return reject(new Error("Cache client is `null`"))
-
+      const start = process.hrtime()
       client.get(key, (err, data) => {
         if (err) return reject(err)
-        if (data) return resolve(JSON.parse(data))
+        if (data) {
+          const end = process.hrtime(start)
+          const time = round(end[1] / 1000000, 3) + (end[0] * 1000)
+          if (time > CACHE_QUERY_LOGGING_THRESHOLD_MS) {
+            warn(`Slow Cache#Get: ${time}ms`)
```

Comment:
Warn is not being logged in production/staging, I’d make this an error for now (not really happy with the log setup overall).

Reply:
I think this should also be done when an error occurs, not just when data is successfully retrieved.